src/renderer/components/node/Node.tsx

```diff
@@ -75,9 +76,10 @@ const NodeInner = memo(({ data, selected }: NodeProps) => {
 
     const { sendToast } = useContext(AlertBoxContext);
     const { categories } = useContext(BackendContext);
+    const { setNodeCollapsed } = useContext(GlobalContext);
     const { getNodeProgress, getNodeStatus } = useContext(ExecutionContext);
 
-    const { id, inputData } = data;
+    const { id, inputData, isCollapsed } = data;
```

Comment:
Instead of assuming that `undefined` is roughly the same `false` below, just set it to `false` here.

```suggestion
    const { id, inputData, isCollapsed = false } = data;
```